src/chemnlp/data_val/config.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+
+from pydantic import BaseModel, Field
+
+
+class Data(BaseModel):
+    datasets: List[str] = Field(default_factory=list)
+    subsample: bool
+    num_train_samples: int
+    num_val_samples: int
+    pad_to_multiple_of: int
+
+
+class Model(BaseModel):
+    base: str
+    name: str
+    revision: str
+
+
+class PromptTune(BaseModel):
+    num_virtual_tokens: int = None
+    prompt_tuning_init_text: str = " "
+
+
+class TrainerConfig(BaseModel):
+    output_dir: str
+    epochs: int = 1
+    learning_rate: float = 3e-4
+    per_device_train_batch_size: int = 32
+    per_device_eval_batch_size: int = 32
+    is_wandb: bool = False
+    wandb_project: str = "chemnlp"
+    run_name: str
```

Comment:
I do not have too strong opinions on that, but might it be useful to also add some validators to the models? For example, a positive learning rate, but smaller than 1.